src/main/java/tictactoe/InputValidator.java

```diff
@@ -61,4 +61,10 @@ private boolean isMarkSelectionInvalid(String input, String otherPlayersMark) {
                 || input.matches(InvalidPlayerMarks.EMPTY.mark)
                 || input.matches(otherPlayersMark);
     }
+
+    private boolean isNotAValidPlayerType(String input) {
+        return (!input.matches(PlayerTypes.HUMAN.type)
```

Comment:
This logic feels like it might work better within the PlayerTypes Enum: So a static method that takes a string and converts it to a PlayerType (also it usually reads better to name the Enum in the singular) - so PlayerType rather than PlayerTypes.

The method might then be `PlayerType.fromString`, which returned a `PlayerType`. You could then either throw an Exception if there was no matching PlayerType for the String or have an INVALID PlayerType (the exception route I think feels more Java-y).

Reply:
Sounds good. I'll change this, thank you